src/components/Sidenav/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Sidenav } from './Sidenav';
+export { SidenavProps } from './Sidenav.types';
+export { SidenavItem } from './SidenavItem';
```

Comment:
Для подкомпонентов нужно экспортить все что есть
```suggestion
export * from './SidenavItem';
```